config/broadcasting.php

```diff
@@ -47,10 +47,10 @@
             'secret' => env('PUSHER_APP_SECRET'),
             'app_id' => env('PUSHER_APP_ID'),
             'options' => [
-                'host' => '127.0.0.1',
-                'port' => 6001,
+                'host' => env('PUSHER_APP_HOST', '127.0.0.1'),
+                'port' => env('PUSHER_APP_PORT', 6001),
                 'cluster' => env('PUSHER_APP_CLUSTER'),
-                'scheme' => 'http',
+                'scheme' => env('PUSHER_APP_SCHEME', 'http'),
```

Comment:
'localpusher' purposefully doesn't implement the environment variables.

The idea is that you use `localpusher` as your `BROADCAST_DRIVER` on the machine that runs the laravel-websockets instance (`php artisan websockets:serve`, via supervisor on production) and client-side still uses your _regular_ pusher configuration, hence `PUSHER_APP_XXX` would still be set to the _external_ parameters, where the _local_ (php-side) takes advantage of a direct http connection to the laravel-websockets server for reduced resource consumption (no SSL layer, it's localhost after all)